.github/workflows/runner-defaults.yml

```
@@ -0,0 +1,116 @@
+# violates the policy. It does NOT fail the build by default — set
+# `fail-on-violation: true` to hard-enforce.
+name: Runner Defaults (reusable)
+
+on:
+  workflow_call:
+    inputs:
+      visibility:
+        description: "Repo visibility: 'public' or 'private'"
+        required: true
+        type: string
+      job-name:
```

actionlint (1.7.12)

[error] "type" is missing at "job-name" input of workflow_call event

(syntax-check)